src/osemgrep/configuring/Engine_type.ml

```diff
@@ -1,3 +1,23 @@
 (* TODO merge with src/core/Engine_kind.ml *)
-type pro_flavor = Language_only | Intrafile | Interfile [@@deriving show]
+
+type analysis_flavor = OSS_intrafile | Deep_intrafile | Deep_interfile
```

Comment:
I think this OSS_intrafile is confusing given we're inside PRO.
Maybe
Intraprocedural | Interprocedural | Interfile would be better.